src/app/data-planes/data/index.spec.ts

```diff
@@ -659,10 +674,21 @@ describe('dataplanes data transformations', () => {
           },
           dataplaneType: 'standard',
           status: 'partially_degraded',
-          unhealthyInbounds: [{
-            port: 1,
-            service: 'service-1',
-          }],
+          unhealthyInbounds: [
```

Comment:
This unhealthyInbouds stuff is a little odd no? Could it just be a funciton on the object?

Reply:
Yeah its very odd.

> Could it just be a funciton on the object?

Exactly, well I plan on just using `inbounds.filter(inbound => !inbound.heath.ready)` whereever we use it instead of having a `unhealthlyInbounds` property.

When we come back to work on this stuff more in earnest I'll be looking to alter this.

Theres a bit of tech debt here to sort out still and I'm still picking my way through it. This PR at least makes unhealthyInbounds have the same properties as a normal inbound.